src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -1054,11 +1056,13 @@
         "3.1.0",
         "3.1.1",
         "3.1.2",
```

Comment:
Should you include BaselineVersion here?

Reply:
A way to check if this matters is to look at any package which should have a reference to this that is being built.  Check if it references GA or the serviced version.

Reply:
It looks like Microsoft.Extensions.Logging.EventSource should reference it, so you can examine it's nuspec.